torchaudio/transforms/_transforms.py

```diff
@@ -2089,3 +2089,58 @@ def forward(
 
         specgram_enhanced.to(dtype)
         return specgram_enhanced
+
+
+class SoudenMVDR(torch.nn.Module):
+    r"""Minimum Variance Distortionless Response (*MVDR* [:footcite:`capon1969high`]) module
+    based on the method proposed by *Souden et, al.* [:footcite:`souden2009optimal`].
+
+    .. devices:: CPU CUDA
+
+    .. properties:: Autograd TorchScript
+
```

Comment:
perhaps something more to introduce/contextualize the equation? it doesn't read very smoothly right now, as it goes from the intro sentence to the compatibility badges and then suddenly to an equation. also, without probing further, i would have thought that `w_{mvdr}(f)` is what this module returns